### PR TITLE
Fix solution build file references (Lombiq Technologies: OCORE-180)

### PR DESCRIPTION
**Pull request by Piedone**
After merging https://github.com/OrchardCMS/OrchardCore/pull/16235 you get the following warning in VS:

image

There aren't any details in the output window apart from a generic "Some of the properties associated with the solution could not be read." but the issue was with the references fixed in this PR.

### CODE REVIEW

**Comment by Piedone** [comment]
@hishamco @MikeAlhayek @sebastienros could you please check this ASAP? Because this affects everyone working on the solution from VS (possibly others).

**Review by MikeAlhayek — APPROVED** [review]
If it fixes the build, why are you awaiting for approval?

**Comment by hishamco** [comment]
Such PR might be exceptional Mike :)

**Comment by Piedone** [comment]
I was waiting for approval because even if I did this small change to the best of my abilities, it can be incorrect, like with any PRs :).

To be clear, I just pinged you three because since this bug is affecting everyone working on OC, I thought it's important to get reviewed ASAP by **any** of you. Not because **all** three of you need to review it. I'll make that clear next time.

**Pull request merged**
